### PR TITLE
Theodore: Correct keyboard mapping and clarify how progs start

### PR DESCRIPTION
**Pull request by felinben**
@Zlika

### CODE REVIEW

File: docs/library/theodore.md [hunk 1]
```diff
@@ -103,17 +107,17 @@ The Theodore core supports the following device type(s) in the controls menu, bo
...
-|----------------------------------------|-----------------------------|
-| ![](../image/retropad/retro_a.png)     | "Fire" button               |
-| ![](../image/retropad/retro_b.png)     | Start button                |
-| ![](../image/retropad/retro_x.png)     | Virtual keyboard: go up     |
-| ![](../image/retropad/retro_y.png)     | Virtual keyboard: go down   |
-| ![](../image/retropad/retro_start.png) | Virtual keyboard: keystroke |
+| RetroPad Inputs                        | User 1 input descriptors       |
+|----------------------------------------|--------------------------------|
+| ![](../image/retropad/retro_a.png)     | "Fire" button                  |
+| ![](../image/retropad/retro_b.png)     | Start screen only: Run content |
```
Comment (Zlika) [inline docs/library/theodore.md]: B button is just really a shortcut for the B letter on the keyboard. If you run the emulator without a disk loaded, start one of the BASIC interpreters and press B on the pad you will just see lots of B letters on the screen. B allows to start "most" of the games (I would say 95%) but not all. For example, the program on a cartridge is generally loaded with the A key (the corresponding entry appears on the main screen only when a cartridge is loaded). Other programs may require the 2nd BASIC to start (C key).

**Comment by felinben** [comment]
just noticed that RA uses f1 (menu toggle), f2 (save state) and f4 (load state). Hotkeys settings will have to reconfigured to make use of those.

**Comment by Zlika** [comment]
RA uses lots of keys as shortcuts. It's not a problem for game consoles but for computers with a full keyboard it is. For example if you type H then the emulator resets. I don't know if there is a cleaner way to manage this.

**Comment by felinben** [comment]
As a workaround there are a couple of overlays for keyboard but it is actually not that ideal for serious typing. 
I have mapped the ‘enable hotkeys’ hotkey in settings > input > input hotkey binds and it behaves weirdly with function keys. Same thing with "game focus toggle".

**Pull request merged**